### ISSUE TITLE
The disabled prop doesn't work with singleton

### ISSUE THREAD

**Issue by praneetrohida** [atomiks/tippyjs-react#247]
I have a usecase where I need to disable a Tooltip which is a singleton target, but I noticed that passing the `disabled` prop as an override doesn't work

**Comment by garretteklof** [comment]
Yeah, I'm also having trouble programmatically disabling a singleton. I'm trying to just disable using `onTrigger` and disabling the instance directly, however I'm noticing a considerable lag (it disables eventually). Any particular pattern you may suggest for this?

**Comment by Slapbox** [comment]
For anyone having this issue, this is the way: https://github.com/atomiks/tippyjs-react/issues/247

**Comment by antontsvil** [comment]
same issue with the `visible` prop it seems

**Comment by Slapbox** [comment]
Does using this syntax resolve the issue? https://github.com/atomiks/tippyjs-react/issues/202#issuecomment-615966879
